src/Constants.js

```diff
@@ -148,39 +172,30 @@ export const majorScaleToMinorScale = (majorScale) => {
 
 export const buildRelativeMinorScales = () => {
     let minorScalesContainer = [];
```

Comment:
You can rewrite as:
`const minorScaledContainer = allMajorScales.map(scale => majorScaleToMinorScale(majorScale))`